specification/cognitiveservices/data-plane/Language/preview/2022-07-01-preview/readme.md

```diff
@@ -0,0 +1,54 @@
+# Cognitive Services Language SDK
+
+This is the AutoRest configuration file the Cognitive Services Language SDK.
+
+> see https://aka.ms/autorest
+
+## Releases
+
+The current preview release of Language is 2022-07-01-preview.
+
+<!-- The current stable release of Language is 2022-05-01. -->
+
+```yaml
+tag: release_2022-07-01-preview
```

Comment:
This file isn't in a standard place. Only the readme under "Language" should be updated. Change the `tag` to this recent version, and add the section below lines 25 through 35 as a new block. Do not update existing releases. The only update should be this tag here I'm commenting on.

Reply:
Thanks - made the change in the Language Readme and edited the tag. Also removed the corresponding Readme in the preview folder.